ref/README.md

```diff
@@ -79,6 +84,27 @@ On many systems, the above will suffice. However, some systems will require you
 find dependencies, particularly if software depencencies are not installed in standard locations.
 See below for more information.
 
+### Building with GPTL profiling support
+
+If GPTL is installed and loaded onto system paths, this kernel may be built with support for
```

Comment:
I suggest inserting a sentence here explaining what GPTL is and does.  As written, the implicit expectation is that the reader already knows what GPTL is and does.